app/src/androidTest/java/org/mozilla/fenix/ui/SettingsPrivacyTest.kt

```diff
@@ -307,6 +308,18 @@ class SettingsPrivacyTest {
             addShortcutName(pageShortcutName)
             clickAddShortcutButton()
             clickAddAutomaticallyButton()
+        }
+
+        mDevice.waitForIdle()
+        // We need to close the existing tab here, to open a different session
+        restartApp(activityTestRule)
+        browserScreen {
+        }.openTabDrawer {
+            closeTab()
+        }
+
+        mDevice.pressHome()
```

Comment:
Wondering if we could add this as a transition

Reply:
@isabelrios changed with another method that already does the home button press. Please re-check.

Reply:
Thanks Oana, the more we can add to robots the better, test looks clearer from my view at least :) hope it make sense